features/enrich/variant-group/edit_variant_group_attribute_value.feature

```diff
@@ -132,7 +132,8 @@ Feature: Editing attribute values of a variant group also updates products
     When I change the "Simple" to "blue"
     And I save the variant group
     Then I should not see the text "There are unsaved changes."
-    When I am on the "boot" product page
+    When I am on "enrich/product/"
```

Comment:
Couldn't find another way. The problem seems to come from `Pim\Behat\Context\NavigationContext::openPage()` called by `When I am on the "foo" product page`.
It calls the `open` method of the behat Sensiolab page extension. This method ends on the same Mink method than `When I am on "enrich/product/"`, except it does a lot of other stuff, which obviously create our problem.
Here, I just replace this call by the original Mink one, which works perfectly fine.